lib/identity_pki_web/controllers/page_controller.ex

```diff
@@ -31,13 +31,24 @@ defmodule IdentityPkiWeb.PageController do
   end
 
   def verify(conn, params) do
-    # hmac = request.headers['HTTP_AUTHENTICATION']
+    authentication_headers = Plug.Conn.get_req_header(conn, "authentication")
```

Comment:
This returns a list because it's technically allowed to have multiple headers with the same key. We return a succed if there's no headers, a failure if there's more than one, and run the validation if there's one (which should be the vast majority of the time).

Reply:
**Question:** Why are we passing if there is no header passed? Shouldn't that also be a failure?

Reply:
I misunderstood the Ruby version, you are correct. It should be a failure if a secret is configured and no HMAC is included. Fixed in 6b76153